packages/web-runtime/src/services/auth/userManager.ts

```diff
@@ -176,18 +176,16 @@ export class UserManager extends OidcUserManager {
       this.store.commit('SET_SETTINGS_VALUES', settings)
 
       role = await this.fetchRole({ graphUser, accessToken, roles })
+    } else {
+      userGroups = await this.clientService.owncloudSdk.users.getUserGroups(login.id)
     }
-    const [user, userGroups] = await Promise.all([
-      await this.clientService.owncloudSdk.users.getUser(login.id),
-      await this.clientService.owncloudSdk.users.getUserGroups(login.id)
-    ])
     this.store.commit('SET_USER', {
       id: login.id,
       uuid: graphUser?.data?.id || '',
       username: login.username || login.id,
       displayname: user.displayname || login['display-name'],
       email: login?.email || user?.email || '',
-      groups: userGroups,
+      groups: graphUser?.data?.memberOf || userGroups,
```

Comment:
Wouldn't it make more sense to set `userGroups = graphUser?.data?.memberOf` inside the `if` block above that deals with graph related requests?

Reply:
we have the same behavior with the uuid, so I liked to keep that.
But NVM, i can change that

Reply:
hmmm... true. also fine then 🙈